content/using/operations/using-your-ship.md

```diff
@@ -570,6 +570,33 @@ Syntax `;unset notify`
 Do not notify when your ship name is mentioned.
 
 
+## S3 {#s3}
+
+Amazon S3, which stands for Amazon Simple Storage Service, is a technology that was introduced by Amazon Web Services for cloud storage that has since been extended more broadly into a [standard programming interface](https://en.wikipedia.org/wiki/Amazon_S3#S3_API_and_competing_services).
+
+Our S3 support lets you store credentials for your S3 object storage buckets on your Urbit ship; once you have done so, you will get additional functionality for uploading your own media within Chat and Groups (for avatars).
+
+### Bucket Setup
+
+If you’re using AWS, it will have to support signature v2, not v4. We've found DigitalOcean to work well also. The bucket has to be publicly readable; allow CORS from `*` origins, and allow `*` headers. Your provider should have details on setting access permissions and CORS.
+
+### Add your credentials: Landscape
+
+Navigate to your `https://<your-ship-url>/~profile/settings`, or click your sigil in the upper-right corner of the window. Fill out the **S3 Credentials** and **S3 Buckets** sections and you're all set!
+
+### Add your credentials: Dojo
+
+Once you have gotten your bucket setup, poke the `s3-store` on your ship with your details. You can do this in Dojo or in web Dojo.
+
+```
+:s3-store|set-endpoint 'endpoint.mys3provider.com’
```

Comment:
Or maybe this is just the domain ... I think it would be good to specify what this is for actual Amazon S3, not just the general case where it could be something else pretending to be S3.

Reply:
The endpoint is indeed the domain. S3 has diff signature versions per endpoint that should be coordinated.

Reply:
Ok.  How do I set the signature version? Does that go into the domain somehow, is it a setting elsewhere in the AWS console, or both?

Reply:
When creating a new bucket I don't even see an option for setting signature version. Should we just remove this instruction @matildepark ?

Reply:
https://docs.aws.amazon.com/general/latest/gr/s3.html